buildspec.yml

```diff
@@ -3,7 +3,7 @@ version: 0.2
 phases:
   install:
     runtime-versions:
-      nodejs: 18
```

Comment:
Still an issue with `Unknown runtime version named '18' of nodejs. This build image has the following versions: 12, 14`